INSTRUCTION.md

```diff
@@ -0,0 +1,24 @@
+# How to validate the changes
+
+#Use this command to run Bootstrap Script
```

Comment:
There is a missing space after the '#' symbol. It should be '# Use this command to run Bootstrap Script' for consistency and readability.